noir-projects/noir-contracts/contracts/test/spam_contract/src/types/token_note.nr

```diff
@@ -1,24 +0,0 @@
-use aztec::{macros::notes::note, protocol_types::{address::AztecAddress, traits::Packable}};
```

Comment:
The `BalanceSet` from token contract uses normal `UintNote` instead of this custom non-sense